src/app/pages/admin/buyer-list/buyer-list.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="buyer-list-container">
+  <div class="flex space-between">
+    <h4 class="text--blue text--center">Buyer list</h4>
+    <mat-form-field class="search-buyer-input">
+      <mat-label>Search buyers</mat-label>
+      <input matInput [(ngModel)]="this.searchText" autocomplete="off" />
+      <mat-icon matSuffix>search</mat-icon>
+    </mat-form-field>
+  </div>
+  <table class="table mt-2">
+    <thead>
+      <tr>
+        <th scope="col" class="title">Buyer name</th>
+        <th scope="col">Login id</th>
+        <th scope="col">Mobile no.</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let buyer of this.appServ.allBuyers | filter: searchText:['name', 'emailId', 'mobNo']">
+        <td>{{ buyer.name }}</td>
+        <td>{{ buyer.emailId }}</td>
+        <td>{{ buyer.mobNo }}</td>
+        <td>
+          <button
+            *ngIf="buyer.approved; else buyerApprovalTemplate"
+            mat-flat-button
+            color="primary"
+            class="rounded-button"
+            [routerLink]="['/buyer', buyer.id, 'bid-list']"
+          >
+            Manage
+          </button>
+          <ng-template #buyerApprovalTemplate>
+            <div class="flex direction-column">
+              <button mat-flat-button color="primary" class="rounded-button" (click)="approveBuyer(buyer)">
+                Approve
+              </button>
+              <button mat-stroked-button color="primary" class="rounded-button mt-2" (click)="denyBuyer(buyer)">
```

Comment:
Can we make this button look `red` and change `Deny` to `Delete user` because thats what we are doing?

Reply:
Since it is an irrevocable action for now, red would be the better option. However, I have a proposition for the name of the button. A simple `Delete` would cause confusion as to whether it is for the buyer or the approval request. I am thinking between `Delete user` and `Delete buyer`, which would be more clear to put forward the action to be taken?

Reply:
`Delete buyer`?

Reply:
Cool :sunglasses:

Reply:
I have added the delete buyer naming and delete user flow following the same.